packages/api/src/period/utils.ts

```diff
@@ -59,17 +59,14 @@ const quantifiersWithCounts = (
   quantifiers: PeriodDetailsQuantifier[]
 ): PeriodDetailsQuantifierDto[] => {
   const quantifiersWithQuantificationCounts = quantifiers.map((q) => {
-    let finishedCount = 0;
-    q.quantifications.forEach((quantification) => {
-      finishedCount = isQuantificationCompleted(quantification)
-        ? finishedCount + 1
-        : finishedCount;
-    });
+    const finishedQuantifications = q.quantifications.filter((quantification) =>
+      isQuantificationCompleted(quantification)
+    );
 
     return {
       _id: q._id,
       praiseCount: q.praiseCount,
-      finishedCount,
+      finishedCount: finishedQuantifications.length,
     };
```

Comment:
i don't mean to be a stickler but what do you think of this code style / implementation? Just for a bit cleaner / more readable code:

```
const finishedCount = q.quantifications.filter((quantification) =>
      isQuantificationCompleted(quantification)
    ).length;

    return {
      _id: q._id,
      praiseCount: q.praiseCount,
      finishedCount,
};
```